### PR TITLE
Fix test and add changelog for 0.7.0

### PR DESCRIPTION
**Pull request by rossf7**

### CODE REVIEW

File: .github/workflows/docker-integration-test.yaml [hunk 1]
```diff
@@ -1,24 +1,27 @@
 name: docker-integration-test
-on: [push]
+on: [pull_request, push]
```
Comment (rossf7) [inline .github/workflows/docker-integration-test.yaml:2]: @mrchrisadams Push doesn't work for forks 🤦‍♂️ now we will run the integration tests once a PR is created.

I also did some code gardening and updated the actions versions.

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+## 0.7.0 2024-06-11
+
+### Changed
+
+- Breaking change to add timestamps to exporter metrics with when provider
+last updated the value.
+- Update Electricity Maps provider to return both estimated and actual values.
+Note: There is a delay before actual values are available.
+- Update location parameter to support multiple locations separated with a
+comma.
```
Comment (rossf7) [inline CHANGELOG.md:26]: @mrchrisadams PTAL 🙏  Do these release notes look OK?

File: .github/workflows/docker-integration-test.yaml [hunk 3]
```diff
@@ -1,24 +1,27 @@
...
   docker-integration-test:
     runs-on: ubuntu-latest
     steps:
       - 
         name: Checkout code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
+      -
+        name: Set up Go
+        uses: actions/setup-go@v5
```
Comment (rossf7) [inline .github/workflows/docker-integration-test.yaml:12]: The go version in the Actions VM is usually pretty old. We used setup-go in the release action. Now we use it in all of them

**Review by mrchrisadams — APPROVED** [review]
LGTM @rossf7, thank you. Merge away.

**Pull request merged**